eagle-server/src/main/webapp/app/dev/public/js/common.js

```diff
@@ -334,6 +336,136 @@
 		return deferred;
 	};
 
+	//==============================Map================================
+	common.map1 = new Map();
+	common.map2 = new Map();
```

Comment:
You can't modify common library for adding `map1` && `map2`. This is common library. Not only for `JMX`